Assets/Scripts/Game/Questing/Place.cs

```diff
@@ -851,6 +854,10 @@ SiteDetails[] CollectQuestSitesOfBuildingType(DFLocation location, DFLocation.Bu
                         // Match building against required type
                         if (buildingSummary[i].BuildingType == buildingType || wildcardFound)
                         {
+                            // Building must not be a player owned house
+                            if (DaggerfallBankManager.IsHouseOwned(buildingSummary[i].buildingKey))
```

Comment:
I think this is a good and perfectly direct way to filter out owned houses from quest selection. There's always that small chance an active quest has already selected this building before player purchases house, but that would be an uncommon edge case and should not affect the operation of most quests, as the markers remain valid for spawns anyway. I'm happy with this approach. 👍